scripts/west_commands/runners/mdb.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020 Synopsys.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''Runner for Synopsys Metaware Debugger(mdb).'''
+
+
+from os import path
+from runners.core import ZephyrBinaryRunner, RunnerCaps
+
+class MdbBinaryRunner(ZephyrBinaryRunner):
+    '''Runner front-end for mdb.'''
+
+    def __init__(self, cfg, cores=1, jtag='digilent', nsim_args='',
```

Comment:
Given we have already quite some parameters for this runner I'd like to have it documented somewhere. IMHO for simpler support it might be done right here. Probably in comments or maybe even in some form of runner's help output (if it's possible, @mbolivar-nordic ?).

And I'd like to not only have description of individual options but also examples of usage for typical cases:
1. UP nSIM session and what's required for that (MDB's .cmd file etc)
2. UP HW target via Digilent connection

Reply:
> Probably in comments or maybe even in some form of runner's help output (if it's possible, @mbolivar-nordic ?).

This is already natively supported by the runners package and west extensions; see e.g. `west flash --context`